openrest4j-api/src/main/java/com/wix/restaurants/orders/OrdersInfo.java

```diff
@@ -98,6 +103,8 @@ public int hashCode() {
     @JsonInclude(Include.NON_NULL)
     public String chargeRoundingStrategy;
 
+    @JsonInclude(Include.NON_NULL)
+    public String orderPoliciesId;

```

Comment:
isn't it supposed to be  orderPolicyId ?

Reply:
orderPolicyId right